.github/workflows/test.yml

```diff
@@ -56,9 +56,14 @@ jobs:
           --env=DRY_RUN=1 --env=GITHUB_REPOSITORY --env=PACKAGECLOUD_REPOSITORY=dev
           --env=PACKAGECLOUD_TOKEN=token --env=PUSH_TO_GIT_REMOTE=origin --env=RUNNER_WORKSPACE
           --volume=/var/run/docker.sock:/var/run/docker.sock docker-builder ${{ matrix.release }}
-          "$PWD"'
+          "cowsay hello" "$PWD"'
```

Comment:
is this intentional?

Reply:
Yep, that's the new parameter with the extra packages. I added these two to verify installation of more than one package (see the last test.yml step), and made sure they were packages which we are very unlikely to ever include in the default image :grin:.